ui/app/AppLayouts/Communities/helpers/TokenObject.qml

```diff
@@ -34,6 +34,7 @@ QtObject {
     property int chainId
     property string chainName
     property string chainIcon
+    property string tokenAddress
```

Comment:
```suggestion
    property string contractAddress
```

Reply:
We have 3 different sources which can be passed to TransferOwnershipPopup (we discussed it): model, TokenObject and json in WalletLayout. If I change property here I would have to change it in the model also which is a bigger change.

Reply:
No, because the TokenObject is passed to TransferOwnershipPopup and "token" property in TransferOwnershipPopup must have tokenAddress field.